docs/userguide/searchspace.md

```diff
@@ -1,46 +1,247 @@
+
+Internally, discrete subspaces are represented by two dataframes, the *experimental* and the *computational* representation.
+
+The experimental representation (`exp_rep`) contains all parameters as they were provided upon the construction of the search space and viewed by the experimenter. The computational representation (`comp_rep`) contains a representation of parameters that is actually used for the internal calculation.
+
+In particular, the computational representation contains no more labels or constant columns. This happens e.g. for [`SubstanceParameter`](baybe.parameters.substance.SubstanceParameter) or [`CategoricalParameter`](baybe.parameters.categorical.CategoricalParameter).
+
+The following example demonstrates the difference:
+```python
+from baybe.parameters import NumericalDiscreteParameter, CategoricalParameter
+from baybe.searchspace import SearchSpace
+
+speed = CategoricalParameter("Speed", values=["slow", "normal", "fast"], encoding="INT")
+temperature = NumericalDiscreteParameter(name="Temperature", values=[90, 105])
+
+searchspace = SearchSpace.from_product(parameters=[speed, temperature])
+```
+
+~~~
+### Experimental representation ###
+     Speed  Temperature
+0    slow         90.0
+1    slow        105.0
+2  normal         90.0
+3  normal        105.0
+4    fast         90.0
+5    fast        105.0
+
+### Computational representation ###
+    Speed  Temperature
+0      0         90.0
+1      0        105.0
+2      1         90.0
+3      1        105.0
+4      2         90.0
+5      2        105.0
+~~~
+
+
+
+## `SubspaceContinuous`
+
+The `SubspaceContinuous` contains all the continuous parameters of a `SearchSpace`. There are different ways of constructing this subspace.
+
+### Using explicit bounds
+
+The [SubspaceContinuous.from_bounds](baybe.searchspace.continuous.SubspaceContinuous.from_bounds) method can be used to easily create a subspace representing a hyperrectangle.
+
+```python
+from baybe.parameters import NumericalContinuousParameter
+from baybe.searchspace import SubspaceContinuous
+
+parameters = [
+    NumericalContinuousParameter("param1", (0, 1)),
+    NumericalContinuousParameter("param2", (-1, 1)),
+]
+bounds = pd.DataFrame({p.name: p.bounds.to_tuple() for p in parameters})
+subspace = continuous = SubspaceContinuous.from_bounds(bounds)
+```
+
+### Constructing from a dataframe
+
+Similar to discrete subspaces, continuous spaces can also be constructed using [`SubspaceContinuous.from_dataframe`](baybe.searchspace.continuous.SubspaceContinuous.from_dataframe).
+However, when using this method to create a continuous space, it will create the smallest axis-aligned hyperrectangle-shaped continuous subspace that contains the points specified in the given dataframe.
+
+```python
+from baybe.parameters import NumericalContinuousParameter
+from baybe.searchspace.continuous import SubspaceContinuous
+
+points = pd.DataFrame(
+    {
+        "param1": [0, 1, 2],
+        "param2": [-1, 0, 1],
```

Comment:
i think this is the best way of visually creating a small dataframe, would you consider also using this style in the other parts eg above?

Reply:
Changed everywhere.